pom.xml

```diff
@@ -10,8 +10,7 @@
 
     <name>cVurl</name>
     <description>
-        cVurl is an open-source java wrapper for an HTTP/2 Client. Written in Java11 and can be used with any
```

Comment:
why removing the first part of the sentence?

Reply:
I have added the first sentence.